shm/include/map.hpp

```diff
@@ -12,7 +12,6 @@ class Map {
     const std::vector<Island>& getAllIslands() { return Islands_; };
     void travel(Island* destination);
     size_t getDistanceToIsland(Island* destination);
-    Island* getCurrentPosition() { return currentPosition_; };
```

Comment:
Why do you remove this line?

Reply:
Line 11: `Island* getCurrentPosition() const { return currentPosition_; };` <--- was duplicated, this version is even better (has const - preferred in getters). Great read -> https://stackoverflow.com/questions/21478342/c-const-in-getter

Reply:
Yes I agree about the const when it comes to getters. I thought that I removed this second getter without const but I didn't. No more questions :)

Reply:
No problem - maybe some mess was created during merging or smth like that.